app/services/model_controller/notifier.rb

```diff
@@ -24,6 +25,18 @@ def success_update(item)
       end
     end
 
+    def success_list_update(item, field_name)
+      respond_to do |format|
+        format.html { redirect_to admin_edit_uri(item) }
+        format.json do
+          field = RademadeAdmin::Model::Graph.instance.model_info(item.class).data_items.data_item(field_name)
+          success_message(item, 'Поле обновлено!', {
+            :display_value => display_item_value(item, field)
+          })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.